packages/javascript/bh-shared-ui/src/components/FinishedJobsTable/FinishedJobsFilter.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+    Button,
+    Dialog,
+    DialogActions,
+    DialogClose,
+    DialogContent,
+    DialogDescription,
+    DialogOverlay,
+    DialogPortal,
+    DialogTitle,
+    DialogTrigger,
+    VisuallyHidden,
+} from '@bloodhoundenterprise/doodleui';
+import { noop } from 'lodash';
+import React, { useState } from 'react';
+import { useObjectState } from '../../hooks';
+import { AppIcon } from '../AppIcon';
+
+type Props = {
+    onConfirm: (filters: Record<string, unknown>) => void;
+};
+
+export const FinishedJobsFilter: React.FC<Props> = () => {
```

Comment:
Added the base layout for the filter dialog. The filter components will come in a separate PR to keep this PR smaller.